package.json

```diff
@@ -23,19 +23,19 @@
     "eslint-plugin-prettier": "^4.1.0",
     "react": "17.0.2",
     "react-native": "0.66.4",
-    "react-native-app-auth": "^6.4.2",
+    "react-native-app-auth": "^6.4.3",
     "react-native-datepicker": "^1.7.2",
     "react-native-element-dropdown": "^2.0.0",
     "react-native-floating-action": "^1.22.0",
     "react-native-gesture-handler": "^2.2.0",
     "react-native-image-picker": "^4.7.1",
+    "react-native-keychain": "^8.1.1",
     "react-native-paper": "^4.12.3",
     "react-native-radio-buttons-group": "^2.2.11",
-    "react-native-reanimated": "^2.2.4",
+    "react-native-reanimated": "^2.2.9",
```

Comment:
1. React Native Reanimated: Reverted to a version compatible with React Native 0.66.5
2. Added pertinent libraries